packages/flutter_tools/test/commands.shard/hermetic/run_test.dart

```diff
@@ -1173,6 +1173,9 @@ class TestRunCommandForUsageValues extends RunCommand {
   // ignore: overridden_fields
   List<Device>? devices;
 
+  @override
+  bool get enableImpeller => false;
```

Comment:
Why was this added?

Reply:
I described this problem in more details [here](https://github.com/flutter/flutter/pull/122184#issuecomment-1460508677). Here's a [test failure](https://github.com/flutter/flutter/runs/11854897675).

I think this was a mistake in all tests in [the usageValues group](https://github.com/flutter/flutter/blob/2b6a79cca5e45ce23cade706bbf729b44b57ef03/packages/flutter_tools/test/commands.shard/hermetic/run_test.dart#L670), but they were passing somewhat by accident because `boolArg` was falling back to `false`:

```dart
bool boolArg(String name) => argResults?[name] as bool? ?? false;
```

Now that we've changed `boolArg` to:

```dart
bool boolArg(String name) => argResults![name] as bool;
```

the tests started throwing, because they (the tests) don't call `Command.run()` so `argResults` are null.

Reply:
ahh, sorry, i missed reading that comment. my fault.

Reply:
pushed a patch that invokes the command first to ensure the arg is passed

Reply:
Thanks, looks cleaner now B)